api-module-library/hubspot/tests/api.test.js

```diff
@@ -1,26 +1,376 @@
-const Manager = require('./manager');
-const mongoose = require('mongoose');
-const config = require('./defaultConfig.json');
+const {Api} = require('../api');
+const config = require('../defaultConfig.json');
+const Authenticator = require('@friggframework/test-environment/Authenticator');
 
-describe(`Should fully test the ${config.label} Manager`, () => {
-    let manager, userManager;
+describe(`${config.label} API tests`, () => {
```

Comment:
Info: isn't using a variable to name part of this block making it harder to recognize this execution in different versions of this repository? 🧐

Reply:
I see what you mean. I don't feel strongly about this either way, but have seen this as a convention in other tests. That said, no reason to continue a bad convention! I imagine the only benefit is convenience of copy/paste test boilerplate, which is a bandaid if not an antipattern.